drivers/adxl345/adxl345.c

```diff
@@ -43,6 +44,12 @@ int adxl345_init(adxl345_t *dev, const adxl345_params_t* params)
     /* get device descriptor */
     dev->params= *params;
```

Comment:
`memcpy(&dev->params, params, sizeof(adx1345_params_t));` was the initial implementation in the previous PR but @aabadie made me change this line. I think copying all parameters with `memcpy` is more clear.